lib/manageiq/messaging/version.rb

```diff
@@ -1,5 +1,5 @@
 module ManageIQ
   module Messaging
-    VERSION = "0.1.6"
+    VERSION = "0.2.0"
```

Comment:
Let's not do thsi change in this PR...we should decide separately the version (more likely it's going to be major as I see something options being dropped, like the ssl/sasl stuff